shell/browser/ui/file_dialog_gtk.cc

```diff
@@ -131,24 +131,17 @@ class FileChooserDialog {
       : parent_(
             static_cast<electron::NativeWindowViews*>(settings.parent_window)),
         filters_(settings.filters) {
-    const char* confirm_text = gtk_util::kOkLabel;
-
-    if (!settings.button_label.empty())
-      confirm_text = settings.button_label.c_str();
-    else if (action == GTK_FILE_CHOOSER_ACTION_SAVE)
-      confirm_text = gtk_util::kSaveLabel;
-    else if (action == GTK_FILE_CHOOSER_ACTION_OPEN)
-      confirm_text = gtk_util::kOpenLabel;
-
     InitGtkFileChooserNativeSupport();
 
+    const char* confirm_text =
+        settings.button_label.empty() ? nullptr : settings.button_label.c_str();
```

Comment:
Can you point to the docs which mention `nullptr` will provide the defaults ? I see chromium continues to provide the labels, ex: https://source.chromium.org/chromium/chromium/src/+/main:ui/gtk/select_file_dialog_impl_gtk.cc;l=500-503;

Reply:
@deepak1556 i can try to find it elsewhere but i referenced: https://source.chromium.org/chromium/chromium/src/+/main:remoting/host/file_transfer/file_chooser_linux.cc;l=86-91?q=gtk_file_chooser_native_new&ss=chromium%2Fchromium%2Fsrc

Reply:
Isn't the api called different in these cases `gtk_file_chooser_native_new` vs our usage of `gtk_file_chooser_dialog_new`

Reply:
@deepak1556 oops, yes, will fix!

Reply:
should be set now.